src/js/reusable-charts/horizontal-bar-chart.js

```diff
@@ -123,6 +123,7 @@ export function horizontalBarChart() {
              */
             barChartSvg.append("g")
                 .attr("class", "grid")
+                .attr("stroke-opacity", 0.2) // fix chart thick lines for downloaded image
```

Comment:
I like the idea but the css also specifies rules for `line` and changes the color as well:
```
.grid, line {
    stroke: #999;
    stroke-opacity: .2;
}
```
can you make sure these are added as well?

Reply:
and can you change the comment? or maybe just remove it.

Reply:
the reason why `stroke-opacity` added as an attribute is that `html2canvas` ignores that CSS rule but it works okay when it added directly as an element attribute.
Probably would be better to remove `stroke-opacity` from CSS and leave it as an attribute, what do you think? @milafrerichs

Reply:
hihi we could remove the css but can you please add the stroke color to the grid as well as to the lines?

Reply:
hm.., actually what we need here is simply change
`.grid, line {
    stroke: #999;
    stroke-opacity: .2;
}`
to
`.grid, line {
    stroke: #999;
}`
and it will work properly (see my latest updates)

Reply:
you're still missing the stroke color for the grids and lines in the resulting image. 
you only changed the stroke-opacity but not the stroke color. 
please add the stroke color as well. (the css is not used in the downloaded image) (I just checked the color codes with screenshots).

Reply:
added, please check latest changes